filebeat/docs/load-balancing.asciidoc

```diff
@@ -21,6 +21,19 @@ output.logstash:
 The `loadbalance` option is available for Redis, Logstash, and Elasticsearch
 outputs. The Kafka output handles load balancing internally.
 
+When `loadbalance: true` is set, Filebeat sends data if-and-only-if it can send
+to all hosts in the list. Filebeat connects to all hosts in the list and evenly
+distributes the events to them. This has the side effect of blocking publishing
+to all hosts if any one of them is down or exerting back-pressure.
+
+When `loadbalance: false` is set, Filebeat picks a host at random from the list
+and sends data, switching over to other list entries in the event of a failure.
+Filebeat also accepts a time to live (TTL), which causes it to voluntarily
+reestablish a connection to a new host after a time even if it is healthy.
+Otherwise, if a TTL is not set, as long as the first chosen endpoint is available, 
+Filebeat continues to communicate with it and only switches to another endpoint
+in case of a failure.
+
```

Comment:
This configuration parameter is not specific to Filebeat it's the same for all Beats and we need to address this issue globally. 

This was already documented separately for each output type:

* Logstash https://www.elastic.co/guide/en/beats/filebeat/current/logstash-output.html#loadbalance
* Redis https://www.elastic.co/guide/en/beats/filebeat/current/redis-output.html#_loadbalance
* Elasticsearch got added 2 weeks ago in https://github.com/elastic/beats/pull/33929

I think we should delete the whole `Load Balancing` section from Filebeat (it's a bit misleading, all beats support the same output parameters) and extend the documentation for each output type if needed.

This is where you can find sources for all the documentation for the supported output types:

* Logstash https://github.com/elastic/beats/blob/main/libbeat/outputs/logstash/docs/logstash.asciidoc
* Redis https://github.com/elastic/beats/blob/main/libbeat/outputs/redis/docs/redis.asciidoc
* Elasticsearch https://github.com/elastic/beats/blob/main/libbeat/outputs/elasticsearch/docs/elasticsearch.asciidoc

After this sources are changed you'd need to run `make update` in the root of the repository or ask an engineer to run it for you. It would re-generate documentation for each beat and apply the change.